components/pd_client/src/lib.rs

```diff
@@ -182,13 +213,19 @@ pub trait PdClient: Send + Sync {
     /// Please note that this method should only be called once.
     fn handle_reconnect<F: Fn() + Sync + Send + 'static>(&self, _: F) {}
 
-    fn get_gc_safe_point(&self) -> PdFuture<u64>;
+    fn get_gc_safe_point(&self) -> PdFuture<u64> {
+        unimplemented!();
+    }
 
     /// Gets store state if it is not a tombstone store.
-    fn get_store_stats(&self, store_id: u64) -> Result<pdpb::StoreStats>;
```

Comment:
Is this refactoring is about `CleanUp`?  If not, shall we break it into another pr?